apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsBooleanControl.tsx

```diff
@@ -24,7 +24,12 @@ export function JsonFormsBooleanControl({
   errors,
   path,
   description,
-}: ControlProps) {
+  schema,
+}: ControlProps): JSX.Element {
+  if (schema.const !== undefined) {
```

Comment:
what's schema.const?

Reply:
`schema.const` is a JSON Schema construct where the particular property takes in a specific literal (can be string/number/boolean). In this case, if the `const` is defined (say it's `true) then it is not meaningful to show the toggle, since the `false` case is invalid.